nttt/tidyup.py

```diff
@@ -2,34 +2,33 @@
 
 import os
 import os.path
-import re
 from pathlib import Path
 
 def fix_meta(src, dst):
-    find_replace(src, dst, "  - \r\n    title:", "  - title:")
+    find_replace(src, dst, "  - \n    title:", "  - title:")
```

Comment:
Removing \r all over the place because the modified implementation of file reading uses \n for any platform.